autowarefoundation/autoware
# [WIP]Feature/decision maker #807

## Status
**DEVELOPMENT/WIP**

## Description
Decision maker recognize the current vehicle condition and decide the control strategy.
Currently, decision maker only support state recognition.
 
![b](https://user-images.githubusercontent.com/1758804/30679566-570ec810-9ed5-11e7-81d7-7150ce935019.gif)
![runtime manager_029](https://user-images.githubusercontent.com/1758804/30679668-f884e9e0-9ed5-11e7-8ce8-bc8421742432.png)


## Status

## Futurework
- Add control/monitoring node based on state
- Add registering way for process handler to "state class" 
- Add to gather object detection result
- Change subscribing topic of waypoint

## Steps to Test or Reproduce

```
1. Start up node like ScreenShot.
2. add jsk's "overlayText" on RVIZ and set "/state/overlay_text" to Topic.
```

state is displayed.
